src/index/DeltaTriples.cpp

```diff
@@ -212,6 +212,18 @@ void DeltaTriplesManager::modify(
   });
 }
 
+// _____________________________________________________________________________
+void DeltaTriplesManager::clear() {
+  deltaTriples_.withWriteLock([this](DeltaTriples& deltaTriples) {
+    deltaTriples.clear();
+    auto newSnapshot = deltaTriples.getSnapshot();
+    currentLocatedTriplesSnapshot_.withWriteLock(
+        [&newSnapshot](auto& currentSnapshot) {
+          currentSnapshot = std::move(newSnapshot);
+        });
+  });
```

Comment:
```suggestion
  modify(&DeltaTriples::clear);
```
(Should work, I hope `std::function` takes function to member pointers, otherwise write the trivial lambda).